test/dataframe.jl

```diff
@@ -405,6 +427,18 @@ module TestDataFrame
     @test findfirst(c -> typeof(c) <: CategoricalVector{Union{Int, Missing}},
                     categorical!(deepcopy(df), 1).columns) == 1
 
+    @testset "categorical!" begin
+        df = DataFrame([["a", "b"], ['a', 'b'], [true, false], 1:2, ["x", "y"]])
+        @test eltypes(categorical!(df)) == [CategoricalArrays.CategoricalString{UInt32},
```

Comment:
Better not hardcode `UInt32` as the default could change in the future. I think in other tests we used another approach based on `isa.(eltypes(...), [CategoricalValue{Char}, ...])`.

Reply:
fixed

Reply:
actually this pattern did not go through so I used `all(map(<:, eltypes(...), [...]))` approach.